interface/globals.php

```diff
@@ -234,6 +235,9 @@
 $twigEnv->addFilter(new Twig_SimpleFilter('translate', function ($string) {
     return xl($string);
 }));
+$twigEnv->addFunction(new Twig_SimpleFunction('setupHeader', function ($asset = null) {
+    Header::setupHeader($asset);
+}));
 
 /** Twig_Loader */
 $GLOBALS['twigLoader'] = $twigLoader;
```

Comment:
Is this twigloader global even needed? It looks like it has been incorporate into the twigEnv above.